# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1506638683 25200
#      Thu Sep 28 15:44:43 2017 -0700
# Node ID e15ad9af158fe333193e88cb7c103b1e23f9102e
# Parent  2cc851e8e03fdbdd1f489a2d686bf86b184bd87f
maint: rename function argument to avoid shadowing class member

* symtab.h (symbol_table::symbol_record::symbol_record_rep::bind_fwd_rep):
Rename function argument to avoid shadowing class member name.  Silences clang
compiler warning -Wshadow.

diff --git a/libinterp/corefcn/symtab.h b/libinterp/corefcn/symtab.h
--- a/libinterp/corefcn/symtab.h
+++ b/libinterp/corefcn/symtab.h
@@ -573,17 +573,17 @@ namespace octave
             }
 
           curr_fcn = fcn;
         }
 
         // We don't forward more than once, so no need to forward the
         // next two.
 
-        void bind_fwd_rep (symbol_record_rep *rep) { m_fwd_rep = rep; }
+        void bind_fwd_rep (symbol_record_rep *fwd_rep) { m_fwd_rep = fwd_rep; }
 
         void unbind_fwd_rep (void) { m_fwd_rep = nullptr; }
 
         symbol_record_rep * dup (scope *new_scope) const;
 
         octave_value dump (void) const;
 
         scope *m_decl_scope;
